cfgov/core/views.py

```diff
@@ -123,3 +126,20 @@ def submit_comment(data):
                              })
 
     return response
+
+
+@csrf_exempt
```

Comment:
I don't know that it would look all the different since you're not really serializing any data or returning anything meaningful, to be honest. It would just mean importing `from rest_framework.decorators.api_view` and replacing `@csrf_exempt` with `@api_view(['POST'])`, unless @vickumar1981 is thinking of doing it differently?